src/EnergyPlus/ChillerElectricEIR.cc

```diff
@@ -1470,7 +1524,7 @@ void ElectricEIRChillerSpecs::size(EnergyPlusData &state)
 
             Real64 rho = FluidProperties::GetDensityGlycol(state,
                                                            state.dataPlnt->PlantLoop(this->CDPlantLoc.loopNum).FluidName,
-                                                           Constant::CWInitConvTemp,
+                                                           this->TempRefCondIn,
```

Comment:
Using a chilled water seemed incorrect since we dealing with a condenser loop. Also, it was inconsistent with the `Cp` below.

Reply:
:+1: